<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">/* ===========================
   QLineEditÔºöÊêúÁ¥¢Ê°Ü editKeyword
   =========================== */
QLineEdit#editKeyword {
    border: 1px solid #888;
    border-radius: 6px;
    padding: 5px 8px;
    background: #f8f8f8;
    selection-background-color: #0080ff;
}

QLineEdit#editKeyword:focus {
    border: 1px solid #409eff;
    background: #ffffff;
}

/* ===========================
   QPushButtonÔºöÊêúÁ¥¢ÊåâÈíÆ btnSearch
   =========================== */
QPushButton#btnSearch {
    background-color: #409eff;
    color: white;
    border-radius: 6px;
    padding: 6px 12px;
}

QPushButton#btnSearch:hover {
    background-color: #66b1ff;
}

QPushButton#btnSearch:pressed {
    background-color: #3a8ee6;
}

/* ===========================
   QListWidgetÔºöÊêúÁ¥¢ÂàóË°® listSearch
   =========================== */
QListWidget#listSearch {
    background: #ffffff;
    border: 1px solid #ccc;
}

QListWidget#listSearch::item {
    border-bottom: 1px solid #e5e5e5;
    padding: 6px;
}

QListWidget#listSearch::item:selected {
    background: #d0e8ff;
    color: #000;
}

/* ===========================
   QLabelÔºöÊ†áÈ¢ò„ÄÅ‰ΩúËÄÖ„ÄÅÂ∞ÅÈù¢
   labelTitle
   labelAuthor
   labelCover
   =========================== */

QLabel{
    font-size: 14px;
    color: #666;
}

/* Ê†áÈ¢ò */
QLabel#labelTitle {
    font-size: 19px;
    font-weight: bold;
    color: #333;
}

/* ÊúÄÊñ∞ */
QLabel#lastchapter {
    font-size: 11px;
    font-weight: bold;
    color: #333;
}
/* Â∞ÅÈù¢ÔºàÂª∫ËÆÆÔºâ */
QLabel#labelCover {
    border: 1px solid #ccc;
    background: #fafafa;
    padding: 4px;
}

/* ===========================
   chapter ÂàóË°® listChapter
   =========================== */
QListWidget#listChapter {
    background: #ffffff;
    border: 1px solid #ccc;
}

QListWidget#listChapter::item {
    padding: 5px;
    border-bottom: 1px solid #eee;
}

QListWidget#listChapter::item:selected {
    background: #d0e8ff;
}

/* ===========================
   QTextBrowserÔºöÂÜÖÂÆπ textContent
   =========================== */
QTextBrowser#textContent {
    background: #fdfdfd;
    border: 1px solid #ccc;
    padding: 8px;
    font-size: 15px;
    color: #333;
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QWidget" name="index_widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: #2c3e50;
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #ffffff,
        stop:1 #f3f3f3
    );
    border: 1px solid #dcdcdc;
    border-radius: 10px;
    padding: 8px 18px;
    font-size: 15px;
    font-weight: 500;
}

QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #ffffff,
        stop:1 #eaeaea
    );
    border-color: #c8c8c8;
}

QPushButton:pressed {
    background: #e0e0e0;
    border-color: #b5b5b5;
}

QPushButton:disabled {
    background: #f7f7f7;
    color: #b0b0b0;
    border-color: #ededed;
}
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="index">
         <property name="text">
          <string>È¶ñÈ°µ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="xuanhuan">
         <property name="text">
          <string>ÁéÑÂπª</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wuxia">
         <property name="text">
          <string>Ê≠¶‰æ†</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dushi">
         <property name="text">
          <string>ÈÉΩÂ∏Ç</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lishi">
         <property name="text">
          <string>ÂéÜÂè≤</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wangyou">
         <property name="text">
          <string>ÁΩëÊ∏∏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="kehuan">
         <property name="text">
          <string>ÁßëÂπª</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nvshen">
         <property name="text">
          <string>Â•≥Áîü</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="book_shelf">
         <property name="text">
          <string>‰π¶Êû∂</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>ÊåâF4ÈöêËóèÁïåÈù¢</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSetApi">
         <property name="text">
          <string>ËÆæÁΩÆ</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="page_widget" native="true">
      <layout class="QHBoxLayout" name="window_widget" stretch="0,0,0">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="search_widget" native="true">
         <property name="maximumSize">
          <size>
           <width>416</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="editKeyword">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>260</width>
                <height>60</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSearch">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>130</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string>üîéÊêúÁ¥¢</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listSearch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="info_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="labelCover">
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>210</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>160</width>
                <height>210</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="info" native="true">
              <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0">
               <item row="0" column="1" colspan="3">
                <widget class="QLabel" name="labelTitle">
                 <property name="text">
                  <string>Â∞èËØ¥</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QLabel" name="lastupdate">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLabel" name="label_4">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>ÊúÄÊñ∞Ôºö</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="label">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Êõ¥Êñ∞Ôºö</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="labelAuthor">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="1" colspan="3">
                <widget class="QLabel" name="label_5">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>5</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>5</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="3">
                <widget class="QLabel" name="full">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="3">
                <widget class="QLabel" name="type">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="label_2">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>‰ΩúËÄÖÔºö</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="2" colspan="2">
                <widget class="QLabel" name="lastchapter">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="5" column="1" colspan="3">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <property name="spacing">
                  <number>12</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="read_btn">
                   <property name="minimumSize">
                    <size>
                     <width>214</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton {
    background-color: #FF5252;  
    color: #FFF8DC;            
    border: none;
    padding: 12px 32px;
    font-size: 16px;
    font-weight: bold;
    border-radius: 5px;
    min-width: 150px;
}

QPushButton:hover {
    background-color: #FF6B6B;
    color: white;
}

QPushButton:pressed {
    background-color: #E53935;
}</string>
                   </property>
                   <property name="text">
                    <string>ÂºÄÂßãÈòÖËØª</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="add_bookshelf">
                   <property name="minimumSize">
                    <size>
                     <width>214</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton {
    background-color: #C9A66B;      /* Â§çÂè§Á∫∏Âº†Ê£ï */
    color: #FFFDF7;
    border: none;
    padding: 12px 32px;
    font-size: 16px;
    font-weight: bold;
    border-radius: 5px;
    min-width: 150px;
}

QPushButton:hover {
    background-color: #D8B782;
    color: white;
}

QPushButton:pressed {
    background-color: #B89155;
}
</string>
                   </property>
                   <property name="text">
                    <string>Âä†ÂÖ•‰π¶Êû∂</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>ÁÆÄ‰ªãÔºö</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="intro">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listChapter"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="prev_btn">
              <property name="text">
               <string>‰∏ä‰∏ÄÈ°µ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="next_btn">
              <property name="text">
               <string>‰∏ã‰∏ÄÈ°µ</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Á¨¨</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edit_page">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>È°µ/ÂÖ±</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="all_page">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>12</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>È°µ</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="chapter_wdiget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QTextBrowser" name="chapter_text">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>550</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QTextBrowser {
    /* ËÉåÊôØÈ¢úËâ≤ÔºöÊüîÂíåÁ±≥ÁôΩ */
    background-color: #F7F3E9;

    /* Ê≠£ÊñáÂ≠ó‰ΩìÔºöÂ∏∏Áî®ÈòÖËØªÂ≠ó‰Ωì */
    font-family: &quot;Microsoft YaHei&quot;, &quot;ÂæÆËΩØÈõÖÈªë&quot;, &quot;SimSun&quot;, &quot;ÂÆã‰Ωì&quot;;

    /* Â≠óÂè∑Â§ß‰∏ÄÁÇπÔºàÈòÖËØªÊõ¥ËàíÊúçÔºâ */
    font-size: 18px;

    /* Â≠ó‰ΩìÈ¢úËâ≤ */
    color: #333333;

    /* Êéß‰ª∂ËæπÊ°ÜÂéªÊéâ */
    border: none;

    /* ÂÜÖËæπË∑ùÔºåÈÅøÂÖçÊñáÂ≠óË¥¥Ëæπ */
    padding: 16px;

    /* Ë°åÈ´òÔºàË°åÈó¥Ë∑ùÔºâ */
    line-height: 1.6;

    /* Ëá™Âä®Êç¢Ë°åÊõ¥Ëá™ÁÑ∂ */
    text-align: left;
}

/* Â§ÑÁêÜÂÜÖÈÉ®ÊÆµËêΩÊ†∑ÂºèÔºàHTMLÊ†áÁ≠æÔºâ */
QTextBrowser p {
    /* È¶ñË°åÁº©Ëøõ‰∏§Â≠óÁ¨¶ */
    text-indent: 2em;

    /* ÊÆµËêΩÈó¥Ë∑ù */
    margin: 0 0 12px 0;
}

/* Ë∂ÖÈìæÊé•Áªü‰∏ÄÈ¢úËâ≤ÔºàÂ¶ÇÊûúÂ∞èËØ¥ÂåÖÂê´ÁõÆÂΩïÈìæÊé•Á≠âÔºâ */
QTextBrowser a {
    color: #1A73E8;
    text-decoration: none;
}

QTextBrowser a:hover {
    text-decoration: underline;
}
</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
